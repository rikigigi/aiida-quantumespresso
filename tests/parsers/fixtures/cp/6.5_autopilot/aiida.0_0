           1  -17.174379109012229        0.0000000000000000        0.0000000000000000     
CG1      -17.1743791090      -17.1743839820      -17.1743840284      -17.1743840283
CG2 1.0000000 0.3000000 0.2734494 0.4999882
           2  -17.174384028256625        0.0000000000000000        0.0000000000000000     
CG1      -17.1743840283      -17.1743841671      -17.1743843595      -17.1743843594
CG2 1.0000000 0.5468989 0.3103654 0.4998714
           3  -17.174384359395894        0.0000000000000000        0.0000000000000000     
CG1      -17.1743843594      -17.1743844164      -17.1743844324      -17.1743844327
CG2 1.0000000 0.6207308 0.4227152 0.5021971
           4  -17.174384432690097        0.0000000000000000        0.0000000000000000     
CG1      -17.1743844327      -17.1743844299      -17.1743844623      -17.1743844635
CG2 1.0000000 0.8454304 0.4132241 0.5196896
           5  -17.174384463460804        0.0000000000000000        0.0000000000000000     
CG1      -17.1743844635      -17.1743844561      -17.1743844711      -17.1743844713
CG2 1.0000000 0.8264481 0.3442926 0.5119051
           6  -17.174384471277826        0.0000000000000000        0.0000000000000000     
CG1      -17.1743844713      -17.1743844720      -17.1743844735      -17.1743844732
CG2 1.0000000 0.6885852 0.3773129 0.4332016
