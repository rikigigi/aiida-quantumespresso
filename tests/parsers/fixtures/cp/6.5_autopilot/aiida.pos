     10  0.00224956
     0.55092489706134E+00     0.89015870069507E+00     0.10589905087797E+01
    -0.38553733103449E+00    -0.43136591600533E+00    -0.14265117462009E+01
    -0.31806693537668E+00    -0.50542901591811E+00     0.41024598358882E+00
     12  0.00239470
     0.55272147992542E+00     0.89175857898354E+00     0.10665465827050E+01
    -0.39465687793746E+00    -0.43040754595054E+00    -0.14304085998303E+01
    -0.31760925012425E+00    -0.50558890643956E+00     0.41001728232033E+00
     15  0.00261240
     0.55613611707348E+00     0.89534221689164E+00     0.10784454410504E+01
    -0.40871477308235E+00    -0.42861229789420E+00    -0.14362163912197E+01
    -0.31694404649945E+00    -0.50592508681234E+00     0.40963659063559E+00
     18  0.00283009
     0.56014821009174E+00     0.89997817285703E+00     0.10902564612484E+01
    -0.42246533537258E+00    -0.42646027338686E+00    -0.14414756310456E+01
    -0.31633539216995E+00    -0.50634933559189E+00     0.40922710436232E+00
     21  0.00304779
     0.56466212507611E+00     0.90542866052216E+00     0.11016191723268E+01
    -0.43553966135389E+00    -0.42400192768186E+00    -0.14459797846778E+01
    -0.31580036648264E+00    -0.50684363767752E+00     0.40879844452194E+00
     24  0.00326549
     0.56962986123640E+00     0.91162427433175E+00     0.11130802417278E+01
    -0.44872716085815E+00    -0.42111528458322E+00    -0.14499465643073E+01
    -0.31528663127364E+00    -0.50741127873428E+00     0.40833005141122E+00
     27  0.00348319
     0.57472227347355E+00     0.91824237617279E+00     0.11241032653183E+01
    -0.46160310288839E+00    -0.41780277023923E+00    -0.14530007428290E+01
    -0.31480016066157E+00    -0.50803194224585E+00     0.40783199859442E+00
     30  0.00370089
     0.57989297403587E+00     0.92507217557905E+00     0.11343356862243E+01
    -0.47379308121592E+00    -0.41410543739851E+00    -0.14550518349146E+01
    -0.31436145580125E+00    -0.50868988801128E+00     0.40732066554315E+00
