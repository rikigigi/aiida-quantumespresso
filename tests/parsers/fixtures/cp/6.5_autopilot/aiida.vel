     10  0.00224956
     0.26693656279184E-03     0.21352115373903E-03     0.12443764452836E-02
    -0.15179329301175E-02     0.14680867077054E-03    -0.65556606791676E-03
     0.78187272957847E-04    -0.22520614031858E-04    -0.36800648585423E-04
     12  0.00239470
     0.33313251636688E-03     0.32140406147391E-03     0.12855157064044E-02
    -0.15363128569730E-02     0.17511611134327E-03    -0.64927985627188E-03
     0.75198771287884E-04    -0.31032510801078E-04    -0.39764740633295E-04
     15  0.00261240
     0.41774170317606E-03     0.46447567732190E-03     0.13365556245295E-02
    -0.15657747288273E-02     0.22178504845686E-03    -0.62653369453936E-03
     0.71752064103199E-04    -0.42891295361167E-04    -0.44376370624359E-04
     18  0.00283009
     0.47217107857843E-03     0.56046488766770E-03     0.12784266987430E-02
    -0.14790811198871E-02     0.25482300140105E-03    -0.53728706494953E-03
     0.62931877581792E-04    -0.50955493066801E-04    -0.46321227112102E-04
     21  0.00304779
     0.53103618535614E-03     0.65056816346121E-03     0.12655668788835E-02
    -0.14519765029224E-02     0.29519972911093E-03    -0.47200396023415E-03
     0.57558769847900E-04    -0.59110493285744E-04    -0.49597682415572E-04
     24  0.00326549
     0.56348933022793E-03     0.71756212329595E-03     0.12647059667939E-02
    -0.14644775909774E-02     0.34587493230258E-03    -0.39783107499650E-03
     0.56311766296833E-04    -0.66464815974913E-04    -0.54179680737341E-04
     27  0.00348319
     0.56796481441584E-03     0.74866813510308E-03     0.11752771758607E-02
    -0.13856036172022E-02     0.38885800105758E-03    -0.27904250531161E-03
     0.51102425174150E-04    -0.71095383510032E-04    -0.56014666909310E-04
     30  0.00370089
     0.58210437743798E-03     0.76675084138047E-03     0.11130803914752E-02
    -0.13432679974678E-02     0.43502844686234E-03    -0.18399118405778E-03
     0.47572726251862E-04    -0.75111205515184E-04    -0.58068075463590E-04
