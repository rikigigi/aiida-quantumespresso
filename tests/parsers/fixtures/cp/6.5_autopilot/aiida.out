
     Program CP v.6.5 starts on 30May2020 at 13:52:21 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     1 processors

     MPI processes distributed on     1 nodes
     Fft bands division:     nmany     =       1
     Reading input from aiida.in
 ADD_RULE: POWER STEERING
   Reading rule:  DT                 3.D0                
 ADD_RULE: POWER STEERING
   Reading rule:  ELECTRON_DYNAMICS  VERLET              
 ADD_RULE: POWER STEERING
   Reading rule:  ORTHOGONALIZATION  ORTHO               
 ADD_RULE: POWER STEERING
   Reading rule:  IPRINT             3                   

   Job Title: WATER


   Atomic Pseudopotentials Parameters
   ----------------------------------

   Reading pseudopotential for specie #  1 from file :
   ../pseudo/H_HSCV_PBE-1.0.upf
   file type is UPF v.1

   Reading pseudopotential for specie #  2 from file :
   ../pseudo/O_HSCV_PBE-1.0.upf
     Message from routine scan_end:
     No INFO block end statement, possibly corrupted file
   file type is UPF v.1


   Main Simulation Parameters (from input)
   ---------------------------------------
   Restart Mode       =      -1   from_scratch   
   Number of MD Steps =      30
   Print out every           10 MD Steps
   Reads from unit    =      50
   Writes to unit     =      51
   MD Simulation time step            =      10.00
   Electronic fictitious mass (emass) =      50.00
   emass cut-off                      =       3.00

   Simulation Cell Parameters (from input)
   external pressure       =            0.00 [KBar]
   wmass (calculated)      =         2493.41 [AU]
   ibrav =    1
   alat  =    10.00000000
   a1    =    10.00000000    0.00000000    0.00000000
   a2    =     0.00000000   10.00000000    0.00000000
   a3    =     0.00000000    0.00000000   10.00000000

   b1    =     0.10000000    0.00000000    0.00000000
   b2    =     0.00000000    0.10000000    0.00000000
   b3    =     0.00000000    0.00000000    0.10000000
   omega =    1000.00000000

    ========================================
    |  CONJUGATE GRADIENT                  |
    ========================================
    | iterations   =           100         |
    | conv_thr     = 0.00000100000 a.u.    |
    | passop       =       0.30000 a.u.    |
    | niter_cg_restart =  20      |
    ========================================

   Energy Cut-offs
   ---------------
   Ecutwfc =   80.0 Ry,      Ecutrho =  320.0 Ry,      Ecuts =  320.0 Ry
   Gcutwfc =   14.2     ,    Gcutrho =   28.5          Gcuts =   28.5
   NOTA BENE: refg, mmx =   0.050000  7680
   Orthog. with Gram-Schmidt
   Electron dynamics with conjugate gradient
   Electron dynamics : the temperature is not controlled
   initial random displacement of el. coordinates with  amplitude=  0.020000

   Electronic states
   -----------------
   Number of Electrons=     8, of States =     4
   Occupation numbers :
   2.00 2.00 2.00 2.00


   Exchange and correlations functionals
   -------------------------------------
     Exchange-correlation=  SLA  PW   PBX  PBC
                           (   1   4   3   4   0   0   0)


   Ions Simulation Parameters
   --------------------------
   Ions are allowed to move
   Ions dynamics with newton equations
   the temperature is computed for     9 degrees of freedom
   ion dynamics with fricp =  0.0000 and greasp =  1.0000
   Ionic position (from input)
   sorted by specie, and converted to real a.u. coordinates
   Species   1 atoms =    2 mass =      1822.89 (a.u.),         1.00 (amu) rcmax =   0.50 (a.u.)
        0.571642     0.943352     0.965650
       -0.243397    -0.435015    -1.378745
   Species   2 atoms =    1 mass =     29166.22 (a.u.),        16.00 (amu) rcmax =   0.50 (a.u.)
       -0.328246    -0.508525     0.413094
   Ionic position read from input file

   All atoms are allowed to move
   Ionic temperature is not controlled


   Cell Dynamics Parameters (from STDIN)
   -------------------------------------
   Starting cell generated from CELLDM
   Constant VOLUME Molecular dynamics
   cell parameters are not allowed to move

   Verbosity: iverbosity =  0



   Simulation dimensions initialization
   ------------------------------------

   unit vectors of full simulation cell
   in real space:                         in reciprocal space (units 2pi/alat):
   1    10.0000    0.0000    0.0000              1.0000    0.0000    0.0000
   2     0.0000   10.0000    0.0000              0.0000    1.0000    0.0000
   3     0.0000    0.0000   10.0000              0.0000    0.0000    1.0000

     G-vector sticks info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Sum        2561    2561    641                96969    96969   12053


   Real Mesh
   ---------
   Global Dimensions   Local  Dimensions   Processor Grid
   .X.   .Y.   .Z.     .X.   .Y.   .Z.     .X.   .Y.   .Z.
    60    60    60      60    60    60       1     1     1
   Array leading dimensions ( nr1x, nr2x, nr3x )   =     60    60    60
   Local number of cell to store the grid ( nrxx ) =     216000
   Number of x-y planes for each processors: 
  |  60,  60  |

   Smooth Real Mesh
   ----------------
   Global Dimensions   Local  Dimensions   Processor Grid
   .X.   .Y.   .Z.     .X.   .Y.   .Z.     .X.   .Y.   .Z.
    60    60    60      60    60    60       1     1     1
   Array leading dimensions ( nr1x, nr2x, nr3x )   =     60    60    60
   Local number of cell to store the grid ( nrxx ) =     216000
   Number of x-y planes for each processors: 
  |  60,  60  |

   Reciprocal Space Mesh
   ---------------------
   Large Mesh
     Global(ngm_g)    MinLocal       MaxLocal      Average
          48485          48485          48485       48485.00
   Smooth Mesh
     Global(ngms_g)   MinLocal       MaxLocal      Average
          48485          48485          48485       48485.00
   Wave function Mesh
     Global(ngw_g)    MinLocal       MaxLocal      Average
           6027           6027           6027        6027.00


   System geometry initialization
   ------------------------------
   ibrav =    1       cell parameters read from input file

   Matrix Multiplication Performances
   ortho mmul, time for parallel driver      =   0.00000 with    1 procs

   Constraints matrixes will be distributed block like on
   ortho sub-group =    1*   1 procs



   Pseudopotentials initialization
   -------------------------------


   Common initialization

   Specie:     1

                        dion 

   Specie:     2
   1  indv=  1   ang. mom=  0

                        dion 
   0.2201

   Cell parameters from input file are used in electron mass preconditioning
   init_tpiba2=    0.39478418

   Short Legend and Physical Units in the Output
   ---------------------------------------------
   NFI    [int]          - step index
   EKINC  [HARTREE A.U.] - kinetic energy of the fictitious electronic dynamics
   TEMPH  [K]            - Temperature of the fictitious cell dynamics
   TEMP   [K]            - Ionic temperature
   ETOT   [HARTREE A.U.] - Scf total energy (Kohn-Sham hamiltonian)
   ENTHAL [HARTREE A.U.] - Enthalpy ( ETOT + P * V )
   ECONS  [HARTREE A.U.] - Enthalpy + kinetic energy of ions and cell
   ECONT  [HARTREE A.U.] - Constant of motion for the CP lagrangian



   Wave Initialization: random initial wave-functions
   Occupation number from init
   nbnd =     4
    2.00 2.00 2.00 2.00
 Randomizing ions velocities according to tempw (INPUT VELOCITIES DISCARDED) 

   formf: eself=    30.31961
   formf:     vps(g=0)=  -0.0007957     rhops(g=0)=  -0.0010000
   formf:     vps(g=0)=  -0.0007862     rhops(g=0)=  -0.0009756
   formf:     vps(g=0)=  -0.0007862     rhops(g=0)=  -0.0009756
   formf:     vps(g=0)=  -0.0007862     rhops(g=0)=  -0.0009756
   formf:     vps(g=0)=  -0.0007776     rhops(g=0)=  -0.0009518
   formf: sum_g vps(g)=  -1.7264663 sum_g rhops(g)=  -0.7188485
   formf:     vps(g=0)=  -0.0037910     rhops(g=0)=  -0.0060000
   formf:     vps(g=0)=  -0.0037674     rhops(g=0)=  -0.0058538
   formf:     vps(g=0)=  -0.0037674     rhops(g=0)=  -0.0058538
   formf:     vps(g=0)=  -0.0037674     rhops(g=0)=  -0.0058538
   formf:     vps(g=0)=  -0.0037505     rhops(g=0)=  -0.0057111
   formf: sum_g vps(g)=  -2.9647389 sum_g rhops(g)=  -4.3130909
   Delta V(G=0):   0.006278Ry,    0.170835eV
 PERFORMING CONJUGATE GRADIENT MINIMIZATION OF EL. STATES

  nfi     ekinc              temph  tempp     etot                 enthal               econs                econt              vnhh    xnhh0   vnhp    xnhp0

       nfi   tempp            E -T.S-mu.nbsp         +K_p #Iter
Step     1     385    -17.175705    -17.175705    -17.167045    36
Step     2     365    -17.175412    -17.175412    -17.170210    10
Step     3     343    -17.175096    -17.175096    -17.170208     7
Step     4     321    -17.174792    -17.174792    -17.170206     7
Step     5     303    -17.174534    -17.174534    -17.170204     7
Step     6     290    -17.174348    -17.174348    -17.170203     7
Step     7     283    -17.174249    -17.174249    -17.170203     7
Step     8     282    -17.174234    -17.174234    -17.170204     7
 ==================================================
  Setting tprint=.true. for this step (last of CG)
 ==================================================
NOTE: eigenvalues are not computed without ortho
Step     9     286    -17.174289    -17.174289    -17.170206     7

 ****************************************************
   Autopilot employ rules: 
    Rule event: iprint                      3
    Rule event: dt                      3.0000
 Rescaling ionic velocities with new dt
 Wow, setting tortho=.true. !
    Rule event: electron_dynamics  VERLET    
 ****************************************************

     10    0.000636106769264    0.0  290.90     -17.174384473231     -17.174384473231     -17.170238901586     -17.169602794817   0.0000   0.0000   0.0000   0.0000
     11    0.000484186490096    0.0  296.25     -17.174281197737     -17.174281197737     -17.170059459617     -17.169575273127   0.0000   0.0000   0.0000   0.0000
     12    0.000202110031247    0.0  307.67     -17.174124147673     -17.174124147673     -17.169739643903     -17.169537533872   0.0000   0.0000   0.0000   0.0000
     13    0.000060578158722    0.0  320.91     -17.174182961994     -17.174182961994     -17.169609787906     -17.169549209747   0.0000   0.0000   0.0000   0.0000
     14    0.000073990161923    0.0  330.74     -17.174375638095     -17.174375638095     -17.169662341715     -17.169588351553   0.0000   0.0000   0.0000   0.0000
     15    0.000078358141209    0.0  333.43     -17.174422008571     -17.174422008571     -17.169670351826     -17.169591993685   0.0000   0.0000   0.0000   0.0000
     16    0.000051269038505    0.0  328.63     -17.174292858534     -17.174292858534     -17.169609731840     -17.169558462802   0.0000   0.0000   0.0000   0.0000
     17    0.000132107237939    0.0  319.37     -17.174226621008     -17.174226621008     -17.169675448763     -17.169543341525   0.0000   0.0000   0.0000   0.0000
     18    0.000343600968717    0.0  310.32     -17.174333676440     -17.174333676440     -17.169911371771     -17.169567770802   0.0000   0.0000   0.0000   0.0000
     19    0.000497212708681    0.0  305.43     -17.174440430763     -17.174440430763     -17.170087829665     -17.169590616956   0.0000   0.0000   0.0000   0.0000
     20    0.000438245427339    0.0  306.28     -17.174380635108     -17.174380635108     -17.170015956083     -17.169577710655   0.0000   0.0000   0.0000   0.0000
     21    0.000249745964890    0.0  311.69     -17.174243291153     -17.174243291153     -17.169801516971     -17.169551771006   0.0000   0.0000   0.0000   0.0000
     22    0.000124133721864    0.0  318.39     -17.174213728626     -17.174213728626     -17.169676503858     -17.169552370136   0.0000   0.0000   0.0000   0.0000
     23    0.000117610722964    0.0  322.48     -17.174289277103     -17.174289277103     -17.169693708217     -17.169576097494   0.0000   0.0000   0.0000   0.0000
     24    0.000137392219671    0.0  321.25     -17.174301295657     -17.174301295657     -17.169723225292     -17.169585833072   0.0000   0.0000   0.0000   0.0000
     25    0.000132257795689    0.0  314.40     -17.174181019516     -17.174181019516     -17.169700596995     -17.169568339199   0.0000   0.0000   0.0000   0.0000
     26    0.000170685209425    0.0  304.01     -17.174055159458     -17.174055159458     -17.169722828192     -17.169552142983   0.0000   0.0000   0.0000   0.0000
     27    0.000298830257222    0.0  293.32     -17.174040985246     -17.174040985246     -17.169860923700     -17.169562093443   0.0000   0.0000   0.0000   0.0000
     28    0.000421827484969    0.0  285.16     -17.174067231850     -17.174067231850     -17.170003466307     -17.169581638822   0.0000   0.0000   0.0000   0.0000
     29    0.000410939446296    0.0  280.81     -17.173992394779     -17.173992394779     -17.169990646210     -17.169579706764   0.0000   0.0000   0.0000   0.0000

 * Physical Quantities at step:    30


                total energy =      -17.17382794235 Hartree a.u.
              kinetic energy =       12.56448 Hartree a.u.
        electrostatic energy =      -23.42788 Hartree a.u.
                         esr =        0.00135 Hartree a.u.
                       eself =       30.31961 Hartree a.u.
      pseudopotential energy =       -4.05754 Hartree a.u.
  n-l pseudopotential energy =        1.95687 Hartree a.u.
 exchange-correlation energy =       -4.20976 Hartree a.u.
           average potential =        0.00000 Hartree a.u.



   Eigenvalues (eV), kp =   1 , spin =  1

  -24.15  -12.57   -8.05   -6.35


   CELL_PARAMETERS
   10.00000000    0.00000000    0.00000000
    0.00000000   10.00000000    0.00000000
    0.00000000    0.00000000   10.00000000

   System Density [g/cm^3] :              0.2017098000


   System Volume [A.U.^3] :           1000.0000000000


   Center of mass square displacement (a.u.):   0.000000

   ATOMIC_POSITIONS
   H       0.57989297403587E+00     0.92507217557905E+00     0.11343356862243E+01
   H      -0.47379308121592E+00    -0.41410543739851E+00    -0.14550518349146E+01
   O      -0.31436145580125E+00    -0.50868988801128E+00     0.40732066554315E+00

   ATOMIC_VELOCITIES
   H       0.58210437743798E-03     0.76675084138047E-03     0.11130803914752E-02
   H      -0.13432679974678E-02     0.43502844686234E-03    -0.18399118405778E-03
   O       0.47572726251862E-04    -0.75111205515184E-04    -0.58068075463590E-04



   Partial temperatures (for each ionic specie) 
   Species  Temp (K)   Mean Square Displacement (a.u.)
        1   4.02E+02     4.4100E-02
        2   3.46E+01     2.2613E-04
     30    0.000282196040857    0.0  279.73     -17.173827942353     -17.173827942353     -17.169841649923     -17.169559453882   0.0000   0.0000   0.0000   0.0000

   writing restart file (with schema): ./out/cp_51.save/
     restart      :      0.00s CPU      0.01s WALL (       1 calls)


   Averaged Physical Quantities
                      accumulated      this run
   ekinc         :        0.00018       0.00018 (AU)
   ekin          :       12.64043      12.64043 (AU)
   epot          :      -31.76551     -31.76551 (AU)
   total energy  :      -17.17438     -17.17438 (AU)
   temperature   :      311.48243     311.48243 (K )
   enthalpy      :      -12.02196     -12.02196 (AU)
   econs         :      -17.16983     -17.16983 (AU)
   pressure      :        0.00000       0.00000 (Gpa)
   volume        :     1000.00000    1000.00000 (AU)



     Called by MAIN_LOOP:
     initialize   :      0.95s CPU      0.96s WALL (       1 calls)
     main_loop    :     30.07s CPU     31.50s WALL (      30 calls)
     cpr_total    :     30.07s CPU     31.50s WALL (       1 calls)

     Called by INIT_RUN:

     Called by CPR:
     cpr_md       :     30.07s CPU     31.50s WALL (      30 calls)
     move_electro :     30.02s CPU     31.45s WALL (      31 calls)

     Called by move_electrons:
     rhoofr       :      3.18s CPU      3.26s WALL (     215 calls)
     vofrho       :     25.45s CPU     26.79s WALL (     215 calls)
     dforce       :      1.06s CPU      1.06s WALL (     248 calls)
     calphi       :      0.01s CPU      0.01s WALL (     123 calls)
     nlfl         :      0.00s CPU      0.00s WALL (      31 calls)

     Called by ortho:
     ortho_iter   :      0.00s CPU      0.00s WALL (      21 calls)
     rsg          :      0.00s CPU      0.00s WALL (      21 calls)
     rhoset       :      0.00s CPU      0.00s WALL (      21 calls)
     sigset       :      0.00s CPU      0.00s WALL (      21 calls)
     tauset       :      0.00s CPU      0.00s WALL (      21 calls)
     ortho        :      0.01s CPU      0.01s WALL (      21 calls)
     updatc       :      0.00s CPU      0.00s WALL (      21 calls)

     Small boxes:

     Low-level routines:
     prefor       :      0.00s CPU      0.00s WALL (     164 calls)
     nlfq         :      0.01s CPU      0.01s WALL (      31 calls)
     nlsm1        :      0.06s CPU      0.06s WALL (     688 calls)
     nlsm2        :      0.01s CPU      0.01s WALL (      31 calls)
     fft          :      6.58s CPU      6.69s WALL (    1721 calls)
     ffts         :      1.54s CPU      1.55s WALL (     430 calls)
     fftw         :      1.89s CPU      1.94s WALL (     926 calls)
     betagx       :      0.31s CPU      0.32s WALL (       1 calls)
     qradx        :      0.00s CPU      0.00s WALL (       1 calls)
     gram         :      0.05s CPU      0.05s WALL (     196 calls)
     nlinit       :      0.92s CPU      0.93s WALL (       1 calls)
     init_dim     :      0.02s CPU      0.02s WALL (       1 calls)
     newnlinit    :      0.00s CPU      0.00s WALL (       1 calls)
     from_scratch :      0.01s CPU      0.01s WALL (       1 calls)
     strucf       :      0.02s CPU      0.02s WALL (      32 calls)
     calbec       :      0.06s CPU      0.06s WALL (     688 calls)


     CP           :     31.07s CPU     32.51s WALL


   This run was terminated on:  13:52:54  30May2020            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
